code/src/components/LastPage.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components/macro'
+import { Background, StartBtn } from './StartPage';
+import { GameCard } from './StyledComponents';
+import castle from '../assets/castle.jpeg';
+
+const LastPage = () => {
+  const username = useSelector((store) => store.game.username);
+  const currentPosition = useSelector((store) => {
+    return store.game.currentPosition;
+  });
+
+  return (
+    <LastClonedBackground>
+      <GameCard title={`Congratulations ${username} you made it!`}>
```

Comment:
Very well done with the UX experience informing that that is the winning page. Like seriously I did not get this directory meant winning the game :D I thought it was game over when I worked with the API in my code which did not make sense :D